<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_perm_Analysis</class>
 <widget class="QMainWindow" name="w_perm_Analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Permutation Analysis</string>
     </attribute>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>101</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>FIGURES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonRunAll">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>116</y>
        <width>70</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>117</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>705</x>
        <y>121</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Short d</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>836</x>
        <y>119</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lond d</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinShortChunksize">
      <property name="geometry">
       <rect>
        <x>705</x>
        <y>169</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Button2PDF">
      <property name="geometry">
       <rect>
        <x>506</x>
        <y>138</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>743</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PARALLELIZATION</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>765</x>
        <y>171</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>chunksize</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Experimental">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>901</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QPushButton" name="ButSelectFile">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>19</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButDelFile">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>20</y>
         <width>112</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFileSelected">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>19</y>
         <width>541</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Button2PNG">
      <property name="geometry">
       <rect>
        <x>567</x>
        <y>138</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);</string>
      </property>
      <property name="text">
       <string>PNG</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinShortProcesses">
      <property name="geometry">
       <rect>
        <x>705</x>
        <y>139</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonClearFigures">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Close All</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>114</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinLongChunksize">
      <property name="geometry">
       <rect>
        <x>833</x>
        <y>169</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>758</x>
        <y>141</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string># processes</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinLongProcesses">
      <property name="geometry">
       <rect>
        <x>833</x>
        <y>139</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QFrame" name="frame_parallel">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>114</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>197</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency Analysis</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableFrequencies">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>901</width>
        <height>281</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string>Interval Name</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>From (Hz)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>To (Hz)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>Short Distance</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>Long Distance</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>Ratio</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>Delta</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>4</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>Theta</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>6</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>10</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>Sigma</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>11</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>16</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>Gamma Low</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>30</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>50</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="8" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="9" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="9" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>131</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of shuffles</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="BoxNumberShuffles">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>128</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>50000</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
      <property name="value">
       <number>5000</number>
      </property>
     </widget>
     <zorder>frame_parallel</zorder>
     <zorder>frame_2</zorder>
     <zorder>label_24</zorder>
     <zorder>ButtonRunAll</zorder>
     <zorder>label_18</zorder>
     <zorder>label_14</zorder>
     <zorder>label_15</zorder>
     <zorder>spinShortChunksize</zorder>
     <zorder>Button2PDF</zorder>
     <zorder>label_17</zorder>
     <zorder>label_16</zorder>
     <zorder>frame_Experimental</zorder>
     <zorder>Button2PNG</zorder>
     <zorder>spinShortProcesses</zorder>
     <zorder>ButtonClearFigures</zorder>
     <zorder>spinLongChunksize</zorder>
     <zorder>label_13</zorder>
     <zorder>spinLongProcesses</zorder>
     <zorder>label_21</zorder>
     <zorder>tableFrequencies</zorder>
     <zorder>label_2</zorder>
     <zorder>BoxNumberShuffles</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
